algopy_testing/tests/utils.py

```diff
@@ -0,0 +1,6 @@
+def prioritize_local_algopy_over_stubs() -> None:
```

Comment:
@daniel-makerx curious to learn if there are better ways to achieve that

Reply:
This shouldn't be required? an `algopy/__init__.py` will take precedence over an `algopy.py`, which is what the stubs provides to keep PyLance happy.

Reply:
This turns out to be rather complex when using "editable" dependencies as the `algopy__init__.py` taking precedence over `algopy.py` only works if the both the `algorand-python` and `algorand-python-testing` packages are colocated in the same venv/path

Solution for now was to make the stubs a dependency only when packaging the testing package but not during develpoment